src/Microsoft.TestPlatform.ObjectModel/Framework.cs

```diff
@@ -43,46 +47,75 @@ private Framework()
         /// <returns>A framework object</returns>
         public static Framework FromString(string frameworkString)
         {
+#if NETSTANDARD1_0
+            var CommonFrameworks = new
+            {
+                Net35 = new { DotNetFrameworkName = Constants.DotNetFramework35, Version = "3.5.0.0" },
+                Net4 = new { DotNetFrameworkName = Constants.DotNetFramework40, Version = "4.0.0.0" },
+                Net45 = new { DotNetFrameworkName = Constants.DotNetFramework45, Version = "4.5.0.0" },
+                NetCoreApp10 = new { DotNetFrameworkName = Constants.DotNetFrameworkCore10, Version = "1.0.0.0" },
+                UAP10 = new { DotNetFrameworkName = Constants.DotNetFrameworkUap10, Version = "10.0.0.0" },
+            };
+#endif
+
             if (string.IsNullOrWhiteSpace(frameworkString))
             {
                 return null;
             }
 
-            NuGetFramework nugetFramework;
+            string name, version;
             try
             {
                 // IDE always sends framework in form of ENUM, which always throws exception
                 // This throws up in first chance exception, refer Bug https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/591142
                 switch (frameworkString.Trim().ToLower())
                 {
                     case "framework35":
-                        nugetFramework = CommonFrameworks.Net35;
+                        name = CommonFrameworks.Net35.DotNetFrameworkName;
+                        version = CommonFrameworks.Net35.Version.ToString();
                         break;
+
                     case "framework40":
-                        nugetFramework = CommonFrameworks.Net4;
+                        name = CommonFrameworks.Net4.DotNetFrameworkName;
+                        version = CommonFrameworks.Net4.Version.ToString();
                         break;
+
                     case "framework45":
-                        nugetFramework = CommonFrameworks.Net45;
+                        name = CommonFrameworks.Net45.DotNetFrameworkName;
+                        version = CommonFrameworks.Net45.Version.ToString();
                         break;
+
                     case "frameworkcore10":
-                        nugetFramework = CommonFrameworks.NetCoreApp10;
+                        name = CommonFrameworks.NetCoreApp10.DotNetFrameworkName;
+                        version = CommonFrameworks.Net35.Version.ToString();
```

Comment:
Why `CommonFrameworks.Net35` is here? Should be `CommonFrameworks.NetCoreApp10`

Reply:
Good catch! Thanks.